tsdb/head.go

```diff
@@ -373,9 +378,17 @@ func NewHead(r prometheus.Registerer, l log.Logger, wal *wal.WAL, opts *HeadOpti
 		opts.SeriesCallback = &noopSeriesLifecycleCallback{}
 	}
 
-	es, err := NewCircularExemplarStorage(opts.NumExemplars, r)
-	if err != nil {
-		return nil, err
+	// ugly hack to get the previous default size
+	if opts.MaxExemplars == 0 {
+		opts.MaxExemplars = 100000
```

Comment:
Should we use `DefaultExemplarsConfig` here to set this?

Reply:
Good point, done